doc/iterations/2023.6/scopes/custom-widgets.adoc

```diff
@@ -0,0 +1,32 @@
+= Scope: Support custom widgets in Form representations
```

Comment:
From my understanding, we were supposed to only work on shapes for the moment and not scopes since they should only live in the PR with the code being developed and merged when the work is done.

Reply:
OK